ocs_ci/ocs/cluster_load.py

```diff
@@ -353,11 +354,22 @@ def get_query(self, query, mute_logs=False):
         """
         now = datetime.now
         timestamp = datetime.timestamp
-        return float(
-            self.prometheus_api.query(
-                query, str(timestamp(now())), mute_logs=mute_logs, log_debug=True
-            )[0]["value"][1]
-        )
+        try:
+            result = float(
+                self.prometheus_api.query(
+                    query, str(timestamp(now())), mute_logs=mute_logs, log_debug=True
+                )[0]["value"][1]
+            )
+        except requests.exceptions.RequestException as e:
+            print(f"Encountered error: {e}. Refreshing connection and retrying...")
```

Comment:
not critical, but better to replace print with log.error ?

Reply:
done